specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -4364,9 +4775,8 @@
           "description": "The status of the invitation.",
           "enum": [
             "Pending",
-            "Accepted",
             "Rejected",
-            "Withdrawn"
```

Comment:
>Withdrawn [](start = 13, length = 9)

Removing an enum is a breaking change. How are you planning to support existing customers/SDK?

Reply:
[discussed]
Not used by customers yet.